packages/react/src/components/UIShell/HeaderMenu.tsx

```diff
@@ -228,9 +257,9 @@ class HeaderMenu extends React.Component {
     };
     const itemClassName = cx({
       [`${prefix}--header__submenu`]: true,
-      [customClassName]: !!customClassName,
+      [customClassName || 'default-class-name']: !!customClassName,
```

Comment:
```suggestion
      [customClassName]: !!customClassName,
```

Reply:
A computed property name must be of type 'string', 'number', 'symbol', or 'any'.ts(2464)

we get this error when we remove the default class name. 

I found another way to resolve this, by enclosing the customClassName in string literal.